cmd/kubernetes_show.go

```diff
@@ -96,53 +96,65 @@ If you wish to use a custom format, the available fields are:
 		} else {
 			ow.WriteKeyValues()
 
-			if kubernetesCluster.UpgradeAvailableTo != "" {
-				fmt.Println()
-				fmt.Printf(utility.Red("* An upgrade to v%s is available. To upgrade use: civo k3s upgrade %s --version %s"), kubernetesCluster.UpgradeAvailableTo, kubernetesCluster.Name, kubernetesCluster.UpgradeAvailableTo)
-				fmt.Println()
+			kubeVersions, err := client.ListAvailableKubernetesVersions()
+			if err != nil {
+				utility.Error("%s", err)
+				os.Exit(1)
 			}
+			if kubernetesCluster.UpgradeAvailableTo != "" {
+				for _, version := range kubeVersions {
+					if version.Type == "deprecated" {
+						continue
+					}
+					if version.Version > kubernetesCluster.Version {
+						fmt.Println()
+						fmt.Printf(utility.Red("* An upgrade to v%s-k3s1 is available. Learn more about how to upgrade: civo k3s upgrade --help"), version.Version)
```

Comment:
We shouldn't be printing this line multiple times. We should create a single string with all the versions to display and then print it out. 

Also, this code is a lot similar to what we're doing in this file, so maybe create a common function and invoke it at both places? https://github.com/civo/cli/blob/master/cmd/kubernetes_list_version.go

Reply:
We can't really create a common function, some bits are different in `kubernetes_list_version.go`. For example, here we list the available versions, check if they're deprecated, and if not all the versions with `stable` and `development` are available for upgrade. While there, we we list the available versions, check if they're deprecated, and then list them using the output writer.

Reply:
I have what you mentioned above regarding creating a single string with all the versions to display and then print it out.